src/reader_lcf.cpp

```diff
@@ -7,6 +7,7 @@
  * file that was distributed with this source code.
  */
 
+#include <cstdio>
```

Comment:
Could you remove the
```cpp
#ifdef NDEBUG
#  include <stdio.h>
#endif
```
that is a few lines below? Is not needed anymore after adding this.

Reply:
Well, NDEBUG wasn't defined by default for the GCW0 build so i removed that line (as you requested) and added NDEBUG to OpenDingux's makefile.